RELEASE/scripts/autoscend/iotms/mr2015.ash

```diff
@@ -1189,10 +1189,6 @@ boolean LX_dinseylandfillFunbucks()
 	{
 		return false;
 	}
-	if(get_property("auto_dinseyGarbageMoney").to_int() == my_daycount())
-	{
-		return false;
-	}
```

Comment:
this was meant to ensure we do not get money more than once a day. However it was not reset to 0 on ascension so it made it so we do not get money ever.
also we have since already added use of mafia tracking to handle it better via `_dinseyGarbageDisposed`